glean-core/python/tests/test_loader.py

```diff
@@ -14,7 +17,9 @@
 
 
 def test_builtin_pings():
-    assert set(dir(_builtins.pings)).issuperset(set(["metrics", "baseline", "events"]))
+    assert set(dir(_builtins.pings)).issuperset(
+        set(["metrics", "baseline", "events", "deletion_request"])
```

Comment:
Hmm... I'm not sure.  Let me just make it equality which will help us catch unexpected changes better in the future.

Reply:
`superset` is needed because the object also contains a bunch of members that exist on every Python object, as well as the reason_code members:

```
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'baseline', 'baseline_reason_codes', 'deletion_request', 'deletion_request_reason_codes', 'events', 'events_reason_codes', 'metrics', 'metrics_reason_codes']
```

Reply:
ugh, in that case it's just a bad test really, as it already missed the `deletion_request` part.
Maybe we need to iterate over that, filter by instances of ping and check that against our exact list?
Let's break that out though.